worldedit-core/src/main/java/com/sk89q/worldedit/internal/wna/WorldNativeAccess.java

```diff
@@ -184,6 +187,10 @@ default void markAndNotifyBlock(NP pos, NC chunk, NBS oldState, NBS newState, Si
 
         // Seems used only for PoI updates
         onBlockStateChange(pos, oldState, blockState1);
+
+        if (sideEffectSet.shouldApply(SideEffect.UPDATE)) {
+            updateBlock(pos, oldState, newState);
+        }
```

Comment:
Should be moved to before `LIGHTING` in `applySideEffects`, otherwise this'll get called on normal set-block too, which is unnecessary.